cypress/plugins/index.js

```diff
@@ -64,18 +66,29 @@ module.exports = async on => {
     },
   });
 
-  // to allows usage of a mock proxy
   on('before:browser:launch', (browser = {}, args) => {
     if (browser.name === 'chrome') {
+      // to allows usage of a mock proxy
       args.push('--ignore-certificate-errors');
 
       return args;
     }
 
     if (browser.name === 'electron') {
+      // to allows usage of a mock proxy
       args['ignore-certificate-errors'] = true;
+      // https://github.com/cypress-io/cypress/issues/2102
```

Comment:
Makes sure screenshots are taken in the same resolution (for visual regression testing).